wix/main.wxs

```diff
@@ -192,9 +191,11 @@
             <ComponentGroupRef Id='Binaries'/>
         </Feature>
 
+        <?if $(sys.BUILDARCH) = x64 ?>
         <Feature Id='VCRedistributable' Title='Visual C++ Runtime' AllowAdvertise='no' Display='hidden' Level='1'>
             <MergeRef Id='VCRedist'/>
         </Feature>
+        <?endif ?>
```

Comment:
Likewise, why the addition of the `BUILDARCH` check here?

Reply:
Added a condition that the VC++ redistribution runtime should not be included on arm64 since it was included for x64.

However, I do not know why the redistribution runtime is included. In my opinion, it seems unnecessary.

Reply:
It looks like that was added in #592, as it's needed to actually run Volta and not always available on the system.

Reply:
I looked into it because I couldn't find a merge module for Arm64, and it seems that it is now deprecated.

>Merge modules (.msm files) for Visual C++ Redistributable files are deprecated. We don't recommend you use them for application deployment. Instead, we recommend central deployment of the Visual C++ Redistributable package.

https://learn.microsoft.com/en-us/cpp/windows/redistributing-visual-cpp-files?view=msvc-170#install-the-redistributable-merge-modules